lstm_regularization/embed_regularize.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
Please include the license from the original [AWD-LSTM repo](https://github.com/salesforce/awd-lstm-lm) for https://github.com/salesforce/awd-lstm-lm/blob/master/embed_regularize.py